GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GVFSLockTests.cs

```diff
@@ -91,6 +87,9 @@ public void LockPreventsRenameOfIndexLockOnTopOfIndex()
             string newFileName,
             uint flags);
 
+        [DllImport("libc", EntryPoint = "rename", SetLastError = true)]
+        private static extern int Rename(string oldPath, string newPath);

```

Comment:
Consider renaming this method to `MacRename`, and similarly `MoveFileEx`->`WindowsMoveFileEx`, or some other similar pattern to make the platform-specific usage more clear. Or see my other comment, at some point we may want to think about a refactor to encapsulate platform-specific calls like we did for the product code.